app/models/work.rb

```diff
@@ -0,0 +1,32 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, through: :votes
+
+  validates :category, presence: true
+  validates :title, presence: true, uniqueness: true
+  validates :creator, presence: true
+
+  def self.top_ten(category)
+    top_works = self.top_voted(category)
+
+    if top_works.length > 10
+      return top_works.first(10)
+    else
+      return top_works
+    end
+  end
+
+  def self.spotlight
+    works = self.all
+
+    spotlight = works.sort_by { |work| work.votes.length }
```

Comment:
Just a note, this works, but it's more efficient to use the database to sort them, using `.order`